### ISSUE TITLE
Synthetic Training for Accurate 3D Human Pose and Shape Estimation in the Wild

### ISSUE THREAD

**Issue by Arata2887** [Arata2887/issue_copy#28]
### 0.どんな論文? /What does this work do?

- https://arxiv.org/pdf/2009.10013v2
- BMVC 2020
- Introduces STRAPS (Synthetic Training for Real Accurate Pose and Shape), which uses synthetic training data to improve the accuracy of body shape predictions.
- Curate and provide a challenging evaluation dataset for monocular human shape estimation, SSP-3D(Sports Shape and Pose 3D).

### 1. 論文のモチベーション / Motivation(s) of the paper / document

> ![Fig1](https://github.com/Arata2887/issue_copy/assets/119403000/c5427f40-151d-4a8e-bfa8-95d4ba0087d2)

- While current datasets contain varied and accurate 3D poses, they all suffer from limited body shape diversity, which greatly hampers the shape prediction accuracy of learning-based approaches.
- Additional drawbacks include: baggy clothing obscuring body shape and data captured in indoor MoCap environments being unrepresentative of in-the-wild images.
- To solve these problems, the author proposed strap and SSP-3D database

### 2. 論文のキーアイデア / Key idea(s) of the paper / document

**STRAPS Framework: **
Utilizes proxy representations like silhouettes and 2D joints as inputs to a shape and pose regression neural network, which is trained with synthetic data.

> ![Fig2](https://github.com/Arata2887/issue_copy/assets/119403000/5c90f21c-b40b-4c41-a9c4-86eec41b9bb7)

 (a)STRAPS Training and Inference
- Synthetic training data is generated by sampling SMPL pose and shape parameters, projecting them into 3D vertices and joints, and forming an input proxy representation (X), which is then augmented (X'). 
- During training, the proxy representation is processed by an encoder (ResNet-18) and regressor to predict pose, shape, and camera parameters. Supervision is applied to SMPL parameters, 3D joints, and 2D joints.
- During inference, detection and segmentation CNNs create the proxy representation from input images.

 (b) SSP-3D Optimisation
- Fitting the SMPL model to multi-frame 2D joints and silhouettes of a subject. 
- Initial parameter estimates are refined through multiple iterations, yielding optimised pose and shape parameters. 
- This ensures consistent shape across frames while allowing pose variation, providing accurate SMPL shape and pose parameters for training and evaluation.

**SSP-3D Dataset: **
a new challenging evaluation dataset, which includes images of sports-persons with a variety of body shapes and corresponding pseudo-ground-truth SMPL parameters.

> ![Fig3](https://github.com/Arata2887/issue_copy/assets/119403000/0c749229-1b74-44ef-a9ec-08c7d1080050)

- **(a)** Shows RGB images, initial pose and shape from VIBE, and optimised SSP-3D labels, illustrating improved accuracy after optimisation.
- **(b)** Scatter plot of 2nd (β2) and 3rd (β3) SMPL shape coefficients, showing SSP-3D's - broader body shape diversity compared to other datasets.
- **(c)** Table of subject numbers and variances for β2 andβ3 in each dataset, highlighting SSP-3D's greater shape diversity.

### 3. その他（実験結果、関連研究、分からなかったところ 等）/ Misc (Experimental results, Related papers, Questions)

**Quantitative results:** Quantitative comparison on SSP-3D, MoVi, Human3.6M (Protocol2) and 3DPW.

> ![Fig5](https://github.com/Arata2887/issue_copy/assets/119403000/c1fb801c-cd7a-47f4-a154-1227f087702e)

**Qualitative results:**  The two Figure below show the results of **Qualitative comparison on SSP-3D** and **Qualitative results on Human3.6M (left), 3DPW (middle) and MoVi (right) **, respectively.

> ![Fig6](https://github.com/Arata2887/issue_copy/assets/119403000/332429e2-42dd-4741-825b-ef391bebf858)

> ![Fig7](https://github.com/Arata2887/issue_copy/assets/119403000/853802ee-b149-412e-9238-b2e916fe0e39)

**Ablation studies:** Figure 4 shows how shape and proxy representation (PR) augmentation improve predictions for non-typical body shapes and enhance robustness against noisy inputs.

>![Fig 4](https://github.com/Arata2887/issue_copy/assets/119403000/73b4ebe1-ba49-4e11-a0e0-fece9e9f4994)

### 4.公開コードやデータセットは? /Public codes or datasets?

**STRAPS: ** https://github.com/akashsengupta1997/STRAPS-3DHumanShapePose
**SSP-3D : **https://github.com/akashsengupta1997/SSP-3D
